examples/kubernetes/connectivity-check/connectivity-check.yaml

```diff
@@ -1,112 +1,147 @@
-# Automatically generated by Makefile. DO NOT EDIT
-apiVersion: v1
-kind: Service
-metadata:
-  name: echo-a
-spec:
-  type: ClusterIP
-  ports:
-  - port: 80
-  selector:
-    name: echo-a
 ---
-apiVersion: apps/v1
-kind: Deployment
 metadata:
   name: echo-a
+  labels:
```

Comment:
nit: we might need to update docs related to connectivity check as there are some name changes, and scope changes.

Can be done later.

Reply:
New commit incoming for this.